namastack-outbox-integration-tests/src/test/kotlin/io/namastack/outbox/UnorderedProcessingIntegrationTest.kt

```diff
@@ -82,28 +85,31 @@ class UnorderedProcessingIntegrationTest {
             }
     }
 
-    // ...existing code...
-
     private fun createRecord(
         recordKey: String,
         payload: String,
-    ): OutboxRecord =
+    ): OutboxRecord<String> =
         outboxRecordRepository.save(
             OutboxRecord
-                .Builder()
-                .recordKey(recordKey)
+                .Builder<String>()
+                .key(recordKey)
                 .payload(payload)
-                .recordType("recordType")
-                .build(clock),
+                .handlerId(
+                    @Suppress("ktlint:standard:max-line-length")
+                    $$"io.namastack.outbox.OrderedProcessingIntegrationTest$TestProcessor#handle(java.lang.Object,io.namastack.outbox.handler.OutboxRecordMetadata)",
```

Comment:
Auto detected error:
The handler ID references `OrderedProcessingIntegrationTest` but this is in `UnorderedProcessingIntegrationTest`. This appears to be a copy-paste error. The handler ID should reference the correct class name to ensure proper handler resolution at runtime.